notebooks/MAST/m_dwarf_flares_hst/m_dwarf_flares_hst.ipynb

```diff
@@ -0,0 +1,1412 @@
+{
```

Comment:
<p>+1 Julia. This is a great way to present your introduction. I would add more links to relevant topics, like M Dwarfs. Perhaps even a cool paper on M Dwarf Habitability?  (a "cool" paper I found: <a href="https://doi.org/10.48550/arXiv.2203.10127" target="_blank">https://doi.org/10.48550/arXiv.2203.10127</a>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>